cni/plugin.go

```diff
@@ -198,3 +200,43 @@ func (plugin *Plugin) UninitializeKeyValueStore() error {
 
 	return nil
 }
+
+// check if safe to remove lockfile
+func (plugin *Plugin) IsSafeToRemoveLock(processName string) bool {
+	if plugin != nil && plugin.Store != nil {
+		// check if get process command supported
+		if cmdErr := platform.GetProcessSupport(); cmdErr != nil {
+			log.Errorf("Get process cmd not supported. Error %v", cmdErr)
+			return false
```

Comment:
Should we return error along with bool? Here we are returning false if there is an error. In such cases if the lock file is left behind, we will never remove it and all the subsequent ADD calls will fail. Instead if we return the error, then we can create an ICM on it indicating that something is wrong and can look at the issue proactively before hitting the customer.

Reply:
ok sure..let me return error also.